src/ansys/mapdl/core/__init__.py

```diff
@@ -4,6 +4,21 @@
 
 import appdirs
 
+# Setup data directory
+try:
+    USER_DATA_PATH = appdirs.user_data_dir(
+        appname="ansys_mapdl_core", appauthor="Ansys"
+    )
+    if not os.path.exists(USER_DATA_PATH):  # pragma: no cover
+        os.makedirs(USER_DATA_PATH)
+
+    EXAMPLES_PATH = os.path.join(USER_DATA_PATH, "examples")
+    if not os.path.exists(EXAMPLES_PATH):  # pragma: no cover
+        os.makedirs(EXAMPLES_PATH)
+
+except:  # pragma: no cover
+    pass
```

Comment:
Let's remove the ``try``, because I don't see a reason for it to fail. And such open ``except`` might hide other things.


```suggestion
# Setup data directory
USER_DATA_PATH = appdirs.user_data_dir(
    appname="ansys_mapdl_core", appauthor="Ansys"
)
if not os.path.exists(USER_DATA_PATH):  # pragma: no cover
    os.makedirs(USER_DATA_PATH)

EXAMPLES_PATH = os.path.join(USER_DATA_PATH, "examples")
if not os.path.exists(EXAMPLES_PATH):  # pragma: no cover
    os.makedirs(EXAMPLES_PATH)

```